### DISCUSSION TITLE
Changing the default embedding model

### DISCUSSION THREAD

**Discussion by PatrickLdA** [microsoft/chat-copilot#988] [Q&A]
Hi!

I'm using this application and making some tests in it integrating with qdrant and one of a few changes I've made was changing the default embedding model to `text-embedding-3-large`.
By simpling switching the embedding model, I've noticed that the retrieval responses from the dabatase now have significantly less correlation than when using the default `ada v2` embedding model.

![image](https://github.com/microsoft/chat-copilot/assets/43734047/60e0d170-a74c-418d-89af-25160dc8dcdc)

Has anyone had this type of problem or made this change? Is there anything I'm forgetting to change so I can enjoy the new embeddings advancements?

Thanks!

**Comment by aaronba** [comment]
Hey @PatrickLdA , can you confirm that the embedding model used to query the index is the same as the embedding mode used to generate the emeddings to store data in the index?  For example if you are using distributed ingestion you may need to make sure that the Kernel  Memory service config has the same embedding model as well.  Just something to check.
